CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# Contributing to the Gamepad Navigator
+
+Thank you for your interest in contributing to this project. As the core values and principles of this project align
+with the [Fluid Project](https://wiki.fluidproject.org/pages/viewpage.action?pageId=3900010), a first good step would
+be to get acquainted with the Fluid Community (see [Get Involved](https://tinyurl.com/yxgwccdr)), and understanding how
+[Infusion](http://docs.fluidproject.org/infusion/) works. The next step should be reviewing the project's
+[documentation](docs) and looking into the [source code](src).
+
+## Code of Conduct
+
+The Fluid Community strives to create new community supports, open governance and recognition systems, and
+collaboration techniques that help make our community more open and welcoming. The Gamepad Navigator and everyone
+participating in it is governed by the [Fluid Community's Code of Conduct](https://tinyurl.com/y5q4c6qy). By
+participating, you are expected to uphold this code. Please report unacceptable behavior on the ["#fluid-work" IRC
+channel](https://tinyurl.com/y53ue7d8) or the [fluid-work mailing list](https://tinyurl.com/y4tpf9ty).
+
+## Process/Workflow
+
+The Gamepad Navigator's [source code](src) is hosted on GitHub. All of the code included in a
+[release](https://github.com/fluid-lab/gamepad-navigator/releases) lives in the `master` branch.
+
+The Gamepad Navigator uses a workflow where individual contributors fork the project repository, work in a branch
+created off of `master`, and submit a pull request against the project repository's `master` branch to merge their
+contributions. See the
+[Coding and Commit Standards](https://wiki.fluidproject.org/display/fluid/Coding+and+Commit+Standards) for more clarity
+on the quality and standards of submitted contributions.
+
+### Open an Issue on GitHub
+
+The Gamepad Navigator uses [GitHub issues](https://github.com/fluid-lab/gamepad-navigator/issues) to keep track of the
```

Comment:
You don't need "the" before open tickets, but I would simplify further, as in "GitHub to track issues".

You also need to add something equivalent to "All work should be associated with a specific GitHub issue."